src/providers/QueryProvider.tsx

```diff
@@ -4,7 +4,7 @@ import { QueryClient, QueryClientProvider } from 'react-query';
 const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
-      staleTime: 3600,
+      staleTime: 60 * 60 * 1000,
```

Comment:
Good catch with the missing ms multiplier. Do you think an hour is a good default though or potentially long for all but the most basic config values?

Reply:
Good question, the default `staleTime` is 5 minutes which is a bit short in my opinion. The data doesn't change that often. 

It does provide a more pleasant browsing experience, especially with URL signing (DRM) since all requests take longer.

Reply:
I thought the signed urls / drm urls only worked for a short time

Reply:
The video DRM tokens are, but that doesn't count for the media/playlist metadata requests as long it's cached. We still do need to invalidate the token after 15 minutes so that we don't risk using invalidated tokens.

Reply:
OK, so the strategy is to use an hour for the default for things that can be cached indefinitely, and then control invalidation for other things manually?